locals.tf

```diff
@@ -8,6 +8,9 @@ data "aws_ami" "al2gpu_ami" {
 }
 
 locals {
+  create_vpc                                = var.vpcId == "" ? true : false

```

Comment:
With the proposed modifications to the vpcId variable's default value of null, this line should look like this:
`create_vpc                                = var.vpcId != null ? true : false`

Reply:
Done

Reply:
Not without some hacking, take a look here:
https://github.com/terraform-docs/terraform-docs/issues/680
https://github.com/terraform-docs/terraform-docs/pull/651

Reply:
@amarin-dspace  then we can leave it as it is. Thanks for checking.